index.d.ts

```diff
@@ -373,7 +373,7 @@ export declare interface TracerOptions {
    * implementation for the runtime. Only change this if you know what you are
    * doing.
    */
-  scope?: 'async_hooks' | 'noop'
+  scope?: 'async_hooks' | 'async_local_storage' | 'noop'
```

Comment:
`async_local_storage` only works with Node 14+. Do you want me to mention that in the comment above? Or is the "Only change this if you know what you are doing" enough?

Reply:
I think it's fine to not provide more explanation for now. We will eventually make this the default in supported versions at which point we'll better document it.

Reply:
Sounds good to me!